sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/HashAggregateExec.scala

```diff
@@ -746,7 +848,7 @@ case class HashAggregateExec(
     val fastRowBuffer = ctx.freshName("fastAggBuffer")
 
     // only have DeclarativeAggregate
```

Comment:
can we copy-paste the comments from https://github.com/apache/spark/pull/20965/files#diff-2eb948516b5beaeb746aadac27fbd5b4R324 ?